class NotFound(Exception):
    pass


class ConcurrencyError(Exception):
    pass
